danmaku/api/desktopTest/TimeBasedDanmakuCollectionTest.kt

```diff
@@ -35,7 +36,11 @@ internal class TimeBasedDanmakuCollectionTest {
                 dummyDanmaku(2.0),
             ),
         )
-        val list = instance.at(flowOf(0.seconds)).events.toList()
+        val list = instance.at(
+            flowOf(0.seconds),
+            danmakuFilterConfig = flowOf(DanmakuFilterConfig()),
```

Comment:
```suggestion
            danmakuFilterConfig = flowOf(DanmakuFilterConfig.Default), 
```

最好提供一个 Default 属性并且一直使用那个属性, 显式表示你要使用的是 default